docs/proposals/feature-coverage.md

```diff
@@ -0,0 +1,173 @@
+# Feature Coverage of the Integration Tests
+
+
+### Imports
+
+#### Where Imported Values Come From
+
+An `Installation` can import values from various objects. An import of type `data` can come from:
+
+- a DataObject
+- the complete data map of a `ConfigMap`
+- one data item of a `ConfigMap`
+- the complete data map of a `Secret`
+- one data item of a `Secret`
+
+An import of type `target` can come from:
+
+- a Target that contains a kubeconfig
+- a Target that contains a Secret ref (not supported by the container deployer)
+
+An import of type `targetList` can come from
+
+- a list of `Target` objects
+- a `TargetListReference` (when a subinstallation gets a `TargetList` from its parent)
+
+
+#### Import Data Mapping
+
+Import data mapping that transforms imported values
+Special case: hard-coded values
+
+Default import data mapping
+
+#### Validating Values against the Import Definition in the Blueprint
+
+Required and optional imports
+
+Type validation
+
+#### Using Imports
+
+Imports can be passed to subinstallations
+
+Imports can be templated into deloy items
+
+Imports can be used in an export data mapping
+
+#### Update Scenarios
+
+Changed import values (data and targets). New values must be passed to subinstallations and deploy items.
+
+Update of imports:
+- adding a new import parameter
+- removing an import parameter
+- importing first from ConfigMap, and then from Secret
+- changing the name of a ConfigMap
+
+
+
+### Exports
+
+Export execution of the blueprint
+
+Export data mapping of the installation
+
+An export value can be written to a `DataObject` or `Target`.
+
+Update scenarios
+
```

Comment:
What about similar things as for imports? Changing the names, types, updating values, sub installations exporting data etc.